GraphQL.py

```diff
@@ -94,7 +94,24 @@ def getBlocks(variables):
   }
 }
 """
-    return (_graphql_request(query, variables))
+    return _graphql_request(query, variables)
+
+
+def getLedgerHash():
+    query = """query {
```

Comment:
We need to add a filter for staking epoch as otherwise this won't be guaranteed to return the right hash. Actually now I think of it, the reason I didn't do this was that if and when there is a hard fork this will start returning the wrong result. Maybe we could also combine a block height in there to guard against that.

Reply:
Something like that?
```
{
  blocks(sortBy: DATETIME_DESC, query: {protocolState: {consensusState: {epoch: 1}}, blockHeight: 7400}, limit: 1) {
    protocolState {
      consensusState {
        stakingEpochData {
          ledger {
            hash
          }
        }
        epoch
      }
    }
  }
}
```